eng/common/scripts/Update-DocsMsToc.ps1

```diff
@@ -175,6 +305,9 @@ $toc = @()
 foreach ($service in $serviceNameList) {
   Write-Host "Building service: $service"
 
+  if ($service -eq "Key Vault") {
```

Comment:
Nit: remember to clean this up before merging 👍 

I recognize debug code :grin: